javascript/ql/src/Security/CWE-116/IncompleteHtmlAttributeSanitization.qhelp

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+	<overview>
+		<p>
+
+			Sanitizing untrusted input for HTML meta-characters is an important
+			technique for preventing cross-site scripting attacks. Usually, this
+			is done by escaping <code>&lt;</code>, <code>&gt;</code>,
+			<code>&amp;</code> and <code>&quot;</code>. But the context in which
+			the sanitized value is used decides which characters that actually
+			need to be sanitized.
+
+		</p>
+
+		<p>
+
+			As a consequence, some programs only sanitize
+			<code>&lt;</code> and <code>&gt;</code> since those are the most
+			common dangerous characters. The lack of sanitization for
+			<code>&quot;</code> is problematic when an incompletely sanitized
+			value is used as an HTML attribute in a string that
+			<strong>later</strong> is parsed as HTML.
```

Comment:
```suggestion
			later is parsed as HTML.
```
Boldface usually comes off a little more aggressive than intended, and this last part of the sentence is really not the central part of the message. Put too much emphasis here and it gives the impression that this vulnerability is somehow related to HTML parsing (as opposed to HTML generation).